src/core/model/types/double_type.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 #include "numeric_type.h"
+#include <boost/math/special_functions/relative_difference.hpp>
```

Comment:
Style guide requires an empty line between different include groups (that is, between local project headers and between library headers)

Reply:
And the order of includes is wrong also